### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by tagener-noisu** [spikecodes/libreddit#863] [Q&A]
I'm using my private instance and it works well, no API limits. Will the project continue to be maintained for us private users?

**Comment by sigaloid** [comment]
I'm the last (semi active?) maintainer, currently behind on a lot of the contributions made.

I'm hoping to at least catch up on the backlog soon. Probably during the last half of December, finals season. Could be time for a new release then.

**Reply by Opening-Button-8988** [reply]
I read this: https://github.com/libreddit/libreddit/issues/785#issuecomment-1624263110

I absolutely love the irony in the fact that companies like Reddit and Youtube lose so little money from tools like yt-dlp, libreddit and ublock that their attempts to block them are so half-assed they continue working like nothing happened.
